lua/nvim-treesitter/parsers.lua

```diff
@@ -1395,7 +1395,7 @@ list.t32 = {
   install_info = {
     url = "https://codeberg.org/xasc/tree-sitter-t32",
     files = { "src/parser.c", "src/scanner.c" },
-    revision = "675cd7de4eb80f2faa1e6d3bc9ee195fa9ef9790",
+    revision = "1dd98248b01e4a3933c1b85b58bab0875e2ba437",
```

Comment:
Is there a reason you're using specific revisions instead of letting the ci auto update it?

Reply:
I am mainly developing the grammar on Linux and don't want to put in the effort to create an own CI agent for Windows or macOS builds. The result is that I broke Windows builds twice in a row without noticing, because I had not been able to add C99-compatible _static_assert()_ correctly.
The pinned revisions are checked more thoroughly to make sure that they build for _nvim-treesitter_ on all expected platforms and that all tests are passed. In addition, by pinning the version, I can work on the grammar without feeling pressured to bring the _nvim-treesitter_ parts back in sync after an auto update.

Please let me know if the _revision_ field should not be used in this way.

Reply:
That's totally fine, I was just curious